modules/apps/jaggery/mdm/src/mdm/modules/user.js

```diff
@@ -540,5 +540,42 @@ var userModule = function () {
         }
     };
 
+    /**
+     * Private method to be used by addUser() to
+     * retrieve secondary user stores.
+     * This needs Authentication since the method access admin services.
+     *
+     * @returns {string array} Array of secondary user stores.
+     */
+    publicMethods.getSecondaryUserStores = function () {
+        var ws = require('ws');
+    	var headers = [{name : "Authorization", value : "Basic YWRtaW46YWRtaW4="}];
```

Comment:
Code lines are not properly aligned it seems. + Can see some hard-coded Authorisation header value in this particular line.